### PR TITLE
Add file based image resolver for rekt tests

### PR DESCRIPTION
**Pull request by creydr**
Since https://github.com/knative-sandbox/reconciler-test/pull/465 we can define a mapping for images used in the rekt tests and our "own" images. This PR adds the mapping for the images used in eventing.

Test run in #123

### CODE REVIEW

**Comment by creydr** [comment]
Needs https://github.com/knative/eventing/pull/6753 to merge first

File: openshift/patches/018-rekt-test-image-changes.patch [hunk 1]
```diff
@@ -45,42 +45,3 @@ index d59abd673..48faff01f 100644
  	//         imagePullPolicy: IfNotPresent
  	//         args:
  	//         - --period=1
```
[the remark this answers is not in the document]
Comment (creydr) [reply openshift/patches/018-rekt-test-image-changes.patch:47]: Yes (didn't see initially, that containersource.go registeres the image too)

**Comment by creydr** [comment]
Tests in #123 with these changes against release-next passed.

/assign @pierDipi @matzew

File: openshift/images.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+knative.dev/eventing/test/test_images/print: registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-print
```
[the remark this answers is not in the document]
Comment (creydr) [reply openshift/images.yaml:1]: We could. But isn't this file relevant for e2e test only anyhow, and before running the tests, the references would be updated? Or am I missing something?

[the remark this answers is not in the document]
Comment (pierDipi) [reply openshift/images.yaml:1]: I was planning to use it in serverless-operator, but we can do that as a follow up

[the remark this answers is not in the document]
Comment (pierDipi) [reply openshift/images.yaml:1]: /lgtm

**Pull request merged**